src/config/rbd/layers.json

```diff
@@ -90,31 +86,38 @@
     "rainfall_agg_month": {
       "title": "1-month rainfall aggregate",
       "type": "wms",
-      "server_layer_name": "rnh_dekad",
+      "server_layer_name": "r1h_dekad",
+      "content_path": "data/myanmar/contents.md#1-month-rainfall-aggregate",
+      "additional_query_params": {
+        "styles": "rfh_1500"
+      },
       "base_url": "https://odc.ovio.org/",
       "date_interval": "days",
-      "opacity": 0.5,
+      "opacity": 0.7,
       "legend_text": "Total aggregate precipitation over a 1-month period, rolling every 10 days. Derived from CHIRPS (UCSB Climate Hazards Group). https://www.chc.ucsb.edu/data/chirps",
       "legend": [
-        { "value": "50", "color": "#faf3d2" },
-        { "value": "100", "color": "#dae3a1" },
-        { "value": "150", "color": "#a0c787" },
-        { "value": "200", "color": "#68ab79" },
-        { "value": "250", "color": "#9beafa" },
-        { "value": "300", "color": "#00b1de" },
-        { "value": "400", "color": "#005ae6" },
-        { "value": "500", "color": "#0000c8" },
-        { "value": "600", "color": "#a000fa" },
-        { "value": ">600", "color": "#fa78fa" }
+        {"value": "-0", "color": "#ffffff", "alpha": 0},
+        {"value": "50", "color": "#faf3d2"},
+        {"value": "100", "color": "#dae3a1"},
+        {"value": "200", "color": "#a0c787"},
+        {"value": "300", "color": "#68ab79"},
+        {"value": "400", "color": "#9beafa"},
+        {"value": "500", "color": "#00b1de"},
+        {"value": "600", "color": "#005ae6"},
+        {"value": "800", "color": "#0000c8"},
+        {"value": "1000", "color": "#a000fa"},
+        {"value": "1500", "color": "#fa78fa"},
+        {"value": "> 1500", "color": "#ffc4ee"}
       ]
     },
     "streak_dry_days": {
       "title": "Longest number of consecutive dry days",
       "type": "wms",
       "server_layer_name": "dlx_dekad",
+      "content_path": "data/myanmar/contents.md#longest-number-of-consecutive-dry-days",
```

Comment:
myanmar?

Reply:
thanks, we'll clean this up later as more documentation on each layer gets produced. hoping to have a central folder for all and custom per country as needed